src/rust/runtime/network/socket/option.rs

```diff
@@ -38,6 +38,9 @@ impl TcpSocketOptions {
 
 impl Default for TcpSocketOptions {
     fn default() -> Self {
-        Self { linger: None }
+        Self {
+            // Setting this to zero by default.
+            linger: Some(Duration::ZERO),
```

Comment:
Shouldn't be the default MSL?

Reply:
Yes, it should be None, but I set it to 0 to avoid needing to set it for every single one of our tests. We should have this as a configuration option, I will add a TODO and a link to my PR #1265

Reply:
So it sounds like a fix to #1265 must come first in the commit history, right?